grammar.js

```diff
@@ -58,7 +58,8 @@ module.exports = grammar({
                         choice(
                             $._statement,
                             $._documentation_brief_container,
-                            $._documentation_tag_container
+                            $._documentation_tag_container,
+                            $.emmy_class
```

Comment:
I am not sure if this should be here but that allows us to do these things:
```lua


---@class TestArray @Numeric table
---@class TestMap @Map-like table

```

Which is cool